### DISCUSSION TITLE
New sticker?

### DISCUSSION THREAD

**Discussion by sawq375** [HasangerGames/suroi#263] [Ideas]
Guys recently due to MatPat from GameTheorists leaving YT can we have a sticker for his memorial
![image](https://github.com/HasangerGames/suroi/assets/97305445/fc1a139e-0abf-4a3a-bbc9-903c615e923a)

**Comment by leia-uwu** [comment]
what

**Comment by juni-purr** [comment]
real

**Comment by sawq375** [comment]
...do you guys not know ITS JUST A THEORY....A GAME THEORY!!!!

**Reply by sawq375** [reply]
so is it gonna get added or not...

**Comment by leia-uwu** [comment]
no, also please don't use github for discussions related with the discord server lmfao

**Reply by sawq375** [reply]
i can't use discord....unfortunately im on a school comp 😔

**Reply by sawq375** [reply]
don't got a personal unblocked comp

**Comment by blazingyoda** [comment]
Both lyz and wizle are continuously repeat offending. It's clear that they need to be banned. They are like mini-ukraineses

**Reply by SovietNuclearFridgeBomb** [reply]
💀

**Reply by sawq375** [reply]
wut does that mean

**Comment by blazingyoda** [comment]
bro its sending me emails for github replys wtf

**Reply by sawq375** [reply]
lol its in your settings to change notifications

**Comment by leia-uwu** [comment]
![sillius](https://github.com/HasangerGames/suroi/assets/69596237/7ee45a95-1bfe-460c-91ca-4828656505f3)
